day_two_guide/topics/increasing_docker_storage.adoc

```diff
@@ -127,6 +141,43 @@ ose-app-node01.example.com   Ready                      24m       v1.6.1+5115d70
 ose-app-node02.example.com   Ready                      24m       v1.6.1+5115d708d7
 ----
 
+*Increasing storage with a new disk*
+
+. Evacuate the node xref:evacuating-a-node[following the steps above].
+
+. Stop the `docker` and `atomic-openshift-node` services:
++
+----
+# systemctl stop docker atomic-openshift-node
+----
+
+. Resize the disk.
+
+. Reattach the disk.
+
+. Run the `docker-storage-setup` command to extend the volume groups and logical
+volumes associated with container storage:
```

Comment:
I'd say:

Verify the /etc/sysconfig/container-storage-setup is configured properly for the new disk (device name, size, etc.), then execute the docker-storage-setup to reconfigure it.